src/View/Helper/ImgUrl.php

```diff
@@ -59,8 +60,12 @@ public function __invoke($relativeName, $filter)
             $format = $binary->getFormat() ?: 'png';
         }
         if ($this->cacheManager->isCachingEnabled($filter, $filterOptions) && $this->cacheManager->cacheExists($relativeName, $filter, $format)) {
+    
+            if($this->cacheManager->useCacheUrl()) {
+                return $this->cacheManager->useCacheUrl() . $this->cacheManager->getCacheUrl($relativeName, $filter, $format);
```

Comment:
How about adding this logic on `HtImgModule\Service\CacheManagerInterface::getCacheUrl` method instead?